src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -38,6 +38,8 @@ class Definition
     private $public = true;
     private $synthetic = false;
     private $abstract = false;
+    private $deprecated = false;
+    private $deprecatedMessage;
```

Comment:
but it would be better to use it rather than having $deprecated be an array IMHO